debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerConnectorTask.java

```diff
@@ -83,12 +83,7 @@ public String version() {
         Offsets<SqlServerPartition, SqlServerOffsetContext> offsets = getPreviousOffsets(
                 new SqlServerPartition.Provider(connectorConfig),
                 new SqlServerOffsetContext.Loader(connectorConfig));
-        SqlServerPartition partition = offsets.getTheOnlyPartition();
```

Comment:
This way, the task will no longer fail if configured with multiple databases. It should use the old API until it supports iteration over partitions.